XiEditor/AppDelegate.swift

```diff
@@ -14,12 +14,16 @@
 
 import Cocoa
 
+//TODO: preferred font should be a user preference
+let defaultFont = CTFontCreateWithName("InconsolataGo" as CFString?, 14, nil)
```

Comment:
Two options:

1. if this is going to be a global, go `FULL_GLOBAL_STYLE`. Otherwise I think it could live comfortably in the app delegate itself, or as a `static` (class) property on `TextDrawingMetrics` (or somewhere else for that matter).